syntax = "proto3";//声明了protobuf的版本

package  mprpcHeader;//声明代码所在位置,对于c++来说是namespace

message rpcHeader
{
    string serviceName  = 1;
    string methodName   = 2;
    uint32 argsSize     = 3;
}